package.json

```diff
@@ -17,26 +17,23 @@
     "Chuck Carpenter <charleswcarpenter3@gmail.com>"
   ],
   "scripts": {
-    "build": "webpack && yarn copy-deps",
+    "build": "yarn clean && webpack --mode production",
+    "postbuild": "rm -f dist/removable-*",
     "clean": "rm -rf dist",
```

Comment:
hadn't you suggested a cross platform option for this?

Reply:
I don't think we need anything cross platform. Everything is generally UNIX based.

Reply:
Closed this PR https://github.com/shipshapecode/shepherd/pull/205 by myself. Adding Git to PATH seems to be a good solution.